api/addons/serializers.py

```diff
@@ -19,6 +19,8 @@ class Meta:
     })
 
     def get_absolute_url(self, obj):
+        if obj['addon'] in ('s3', ):
```

Comment:
Because S3 is currently the only addon that doesn't have nested selectable ~~buckets~~ folders. This prevents a `children` link from appearing, as following it would exclusively result in either empty data or an error.